Vikta-API-testing-sample/src/main/java/com/test/api/dto/ImageItemDTO.java

```diff
@@ -0,0 +1,36 @@
+package com.test.api.dto;
+
+import lombok.Data;
+
+import java.util.List;
+import java.util.Set;
+
+@Data
+public class ImageItemDTO {
+    public ImageItemDTO(String author, Set<Long> categoryIds, String description, String pathToImage, double price, int rating, List<String> tags, String title) {
```

Comment:
you can use lombok annotation @AllArgsConstructor for this. there are couple constructor annotations in lombok worth checking out